_includes/pair-programming.md

```diff
@@ -0,0 +1,60 @@
+# Pair programming
+
+## How to pair
+
+In pair programming, two programmers work together at one workstation.
+The **driver** types at the keyboard while the **navigator** directs
+the driver. Even though the driver is at the keys, their role is the
+more passive; the navigator directs the driver on what to type. The
+driver executes their commands.
+
+The foundation of pair programming is communication. Partners must
+always be communicating throughout. The navigator directs the driver,
+but the driver will often need clarification about what the navigator
+intends for them to do. When in disagreement, the pair needs to
+discuss the proper course of action.
+
+**ABS**: Always be switching!
+
+## Benefits of pair programming
+
+There are a number of reasons we pair:
```

Comment:
Here you should emphasize more that we we do pairing in class and that's why it'll happen in the interview. Then we can go into why we do it in class.